mcs_benchmark_data/benchmark_submission_extractor.py

```diff
@@ -0,0 +1,24 @@
+from mcs_benchmark_data._extractor import _Extractor
+
+from mcs_benchmark_data.path import DATA_DIR_PATH
+
+from mcs_benchmark_data._benchmark_file_names import (
+    _BenchmarkFileNames,
+)
+
+
+class BenchmarkSubmissionExtractor(_Extractor):
+    def __init__(
+        self,
+        file_names: _BenchmarkFileNames,

```

Comment:
_BenchmarkSubmissionFileNames?

Reply:
There is not a _BenchmarkSubmissionFileNames model, because it would only be the same as _BenchmarkFileNames with the metadata file. Should I make a _BenchmarkSubmissionFileNames model and change the name of metadata to submission_metadata or something similar?

Reply:
No, not necessary. Presumably the per (benchmark, submission) combination FileNames subclass will have what's needed.